packages/ndla-ui/src/locale/messages-nb.ts

```diff
@@ -1165,6 +1165,18 @@ const messages = {
         },
       },
     },
+    arena: {
+      title: 'Arena',
+      notification: {
+        title: 'Varslinger',
+        showAll: 'Se alle varslinger',
+        description:
+          'Velkommen til NDLAs Arena. Her kan du diskutere, dele og samarbeide med andre lærere fra hele Norge. Du finner forum for ulike fag og tema.',
```

Comment:
Er denne teksten fra design?
```suggestion
          'Velkommen til NDLA Arena. Her kan du diskutere, dele og samarbeide med andre lærere fra hele Norge. Du finner forum for ulike fag og tema.',
```